releasenotes/notes/tranform-API-da38b196c1c72d44.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    It's now possible to made some transformation to the stored timeseries
+    during the retrieval of the measures. The allowed transformations are
+    absolute and negative.
```

Comment:
No, it's  a "x -1", to have the reverse of absolute, you need to use absolute:negative